includes/gameover.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "constants.h"
+
+class GameOver
```

Comment:
I suggest you have a abstract base class for all these menu object for standardization.